src/main/java/frc/robot/subsystems/vision/NoteCamera.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.subsystems.vision;
+
+
+import org.littletonrobotics.junction.Logger;
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+public class NoteCamera {
+
+  PhotonCamera camera;
+  String path;
+
+  private final double MIN_AREA = 10;
+
+  public NoteCamera(String name) {
+    // This is a constructor
+    camera = new PhotonCamera(name);
+    path = "Vision/" + name.replace("_", "");
+  }
+
+  public PhotonTrackedTarget getTarget() {
+    if(1==1) return null;
+    var res = camera.getLatestResult();
+    int numTargets = res.getTargets().size();
+    Logger.recordOutput(path + "/Valid", numTargets > 0);
+    Logger.recordOutput(path + "/NumTargets", numTargets);
+
+    var best = numTargets > 0 ? res.getTargets().get(0) : null;
+
+    if(best != null) {
+      Logger.recordOutput(path + "/Note", best.getBestCameraToTarget());
```

Comment:
place all strings into constants, if used elsewhere, make global and move to commons folder